internal/api/shared/dependency/provisioning.go

```diff
@@ -74,20 +75,20 @@ func (p *provisioning) applyEntity(entities []modelAPI.Entity) {
 		}
 
 		// retrieve if exists the entity from the Perses API
-		_, apiError := svc.Get(param)
+		_, apiError := svc.Get(param, &crypto.JWTCustomClaims{}) // TODO: fix
```

Comment:
ah yeah that would be great to still be able to inject data without the permission system. Perhaps we can use an `admin` role or something like that?

Reply:
Shouldl not be necessary with Toolbox approach